test/unit/math/prim/mat/fun/autocovariance_test.cpp

```diff
@@ -33,3 +34,33 @@ TEST(ProbAutocovariance, test1) {
   EXPECT_NEAR(1.10, ac[4], 0.01);
   EXPECT_NEAR(0.90, ac[5], 0.01);
 }
+
+TEST(ProbAutocovariance, test2) {
+  // ar1.csv generated in R with
+  //   > x[1] <- rnorm(1, 0, 1)
+  //   > for (n in 2:1000) x[n] <- rnorm(1, 0.8 * x[n-1], 1)
+  std::fstream f("test/unit/math/prim/mat/fun/ar1.csv");
+  Eigen::VectorXd y(1000);
+  for (size_t i = 0; i < 1000; ++i) {
+    double temp;
+    f >> temp;
+    y(i) = temp;
+  }
+
+  // 10K 1K-length AC in 2.9s with g++ -O3 on Bob's Macbook Air
+  Eigen::VectorXd ac(1000);
+
+  size_t ITS = 1;  // only need one for test
```

Comment:
Remove the loop as it's hardcoded to a single iteration.  We're still working through external performance testing, but when people update algorithms, they'll need to write more extensive tests than this.